packages/time-oracle/src/lib.rs

```diff
@@ -11,4 +11,10 @@ pub mod migrate_v1;
 pub enum AlarmError {
     #[error("{0}")]
     Std(#[from] StdError),
+
+    #[error("[Time Oracle] Alarms delivery queue is empty but received reply status")]
+    ReplyOnEmptyAlarmQueue,
+
+    #[error("[Time Oracle] Alarms delivery queue is not empty! Assertion failed")]
+    NonEmptyAlarmQueue,
```

Comment:
Suggest defining just plain two errors, EmptyAlarmsInDeliveryQueue and NonEmptyAlarmsInDeliveryQueue. 
The particular cause we shall provide as String, for example: "no alarms in delivery but received a reply"